rs_description/urdf/rs080n_macro.xacro

```diff
@@ -41,26 +38,26 @@
     <xacro:property name="j5_length" value="0.165"/>
     
     <!-- joint limits [rad] -->
-    <xacro:property name="j1_lower_limit" value="${-pi}"/>
-    <xacro:property name="j1_upper_limit" value="${pi}"/>
-    <xacro:property name="j2_lower_limit" value="${-pi*105/180}"/>
-    <xacro:property name="j2_upper_limit" value="${pi*140/180}"/>
-    <xacro:property name="j3_lower_limit" value="${-pi*155/180}"/>
-    <xacro:property name="j3_upper_limit" value="${pi*135/180}"/>
-    <xacro:property name="j4_lower_limit" value="${-pi*360/180}"/>
-    <xacro:property name="j4_upper_limit" value="${pi*360/180}"/>
-    <xacro:property name="j5_lower_limit" value="${-pi*145/180}"/>
-    <xacro:property name="j5_upper_limit" value="${pi*145/180}"/>
-    <xacro:property name="j6_lower_limit" value="${-pi*360/180}"/>
-    <xacro:property name="j6_upper_limit" value="${pi*360/180}"/>
+    <xacro:property name="j1_lower_limit" value="${radians(-180)}"/>
+    <xacro:property name="j1_upper_limit" value="${radians( 180)}"/>
+    <xacro:property name="j2_lower_limit" value="${radians(-105)}"/>
+    <xacro:property name="j2_upper_limit" value="${radians( 150)}"/>
```

Comment:
`j2_upper_limit` has changed from 140 to 150
```suggestion
    <xacro:property name="j2_upper_limit" value="${radians( 140)}"/>
```

Reply:
Thank you for PR.
I found one mistake and suggested fix.
Please checked it.

Reply:
Thanks. Looks like copy-pasta.